blablas/ep24/index.md

```diff
@@ -0,0 +1,90 @@
+---
+date: 2019-12-29
+time: 21h
+duration: "2:18:03"
+title: "Geeksblabla MSS with Yassine El Kachchani"
+tags: ["startup", "side hustle", "morocco"]
+isNext: false
+video: "2770524023008256"
+published: true
+url: "https://www.facebook.com/sfoukahi/videos/2770524023008256/"
+podcastUrl: "https://www.facebook.com/sfoukahi/videos/2770524023008256/"
+---
+
+In this episode of GeeksBlabla, we discuss startuping with Yassine, the CEO of United Remote.
+
+## Guests
+
+[Yassine El Kachchani](https://www.facebook.com/kachchani)
+
+[Mohammed Aboullaite](https://www.facebook.com/aboullaite)
+
+## Notes
+
+If you want contribute in Geksblabla project by adding the notes check [this issue](https://github.com/DevC-Casa/geeksblabla.com/issues/23/)
+
+0:03 - Who is Yassine El Kachchani ?
+
+0:06 - Were you interested in entrepreneurship as a student ?
+
+0:10 - What lessons did you learn from your very first company ?
+
+0:12 - Brief talk about Lacarteplz.ma
+
+0:16 - How did you come up with the idea of Hidden Founders after two failed companies ?
+
+0:22 - Can you talk a bit about the success of Hidden Founders ?
+
+0:30 - Why did you kill Hidden Founders in favour of United Remote despite the former's success ?
+
+0:34 - Does 3% equity translate to 3% of profit or to 3% of shares ?
+
+0:35 - What do you mean by a tiny project ? An MVP, a landing page or what exactly ?
+
+0:37 - Are there Moroccan incubators which offer training or financing that you can recommend ?
```

Comment:
that offer*

Reply:
[Done](https://github.com/DevC-Casa/geeksblabla.com/commit/e391833e02fee3ee8151c297b56db24b4c62b648#diff-061138f907d11d2472101c5ef1450c5fR40) :heavy_check_mark: